public/components/incontext_insight/index.tsx

```diff
@@ -195,19 +348,36 @@ export const IncontextInsight = ({ children }: IncontextInsightProps) => {
     </>
   );
 
-  const ChatPopoverBody: React.FC<{}> = ({}) => (
-    <EuiFlexGroup>
+  const [userQuestion, setUserQuestion] = useState('');
```

Comment:
Shall we move this `useState` inside `ChatPopoverBody`? Seems this state only been used in this component.

Reply:
Done